Expecto/Expecto.fs

```diff
@@ -544,8 +527,12 @@ module Impl =
 
     fun (printers: TestPrinters) map ->
 
+      let log test =
+        if test.sequenced then Async.StartImmediate else Async.Start
```

Comment:
It's really just the join at the end of the run that should decide the method with which logs are awaited... Just because we sequence tests, we don't have to start their logs immediately. Also, `logger.infoWithBP` will only back-pressure the logger's buffer, not the printing. So the returned `Async<unit>` from such a function call will yield as soon as the buffer has accepted the message – a final flush of the logging infrastructure at the end should then be enough.